containers/Makefile

```diff
@@ -112,8 +112,14 @@ build/%: %
 	cp --archive --parents "$<" build/
 
 # Templates
-build/%: %.j2 config.yml ../tools/render-jinja-template.sh
-	../tools/render-jinja-template.sh config.yml "$<" "$@"
+build/%: %.j2 build/vars-merged-with-config.yml ../tools/render-jinja-template.sh
+	../tools/render-jinja-template.sh build/vars-merged-with-config.yml "$<" "$@"
+
+build/vars-merged-with-config.yml: versions.yml config.yml ../tools/merge-yaml.py
```

Comment:
Please call it `config-with-versions.yml`. You're not using `vars.yml` here.

Reply:
Done.